canonical-web-and-design/cn.ubuntu.com
# Update marketo endpoint

## Done

- Post forms to `https://ubuntu.com/marketo/submit`
- Decapitalise input names 
- Update hidden inputs

## QA
1. Go to https://cn-ubuntu-com-594.demos.haus/contact
2. Fill in the form and submit it 
3. Check it's submitted and redirect to a thank you page on cn.unbuntu.com 
4. Check the submitted data is on the list in marketo 

## Issue / Card

Fixes [#4734](https://github.com/canonical-web-and-design/web-squad/issues/4734)
